.github/workflows/Construct.yml

```diff
@@ -0,0 +1,49 @@
+name: Build installers
```

Comment:
Why is this a separated workflow instead of an additional job in Build?

Reply:
Not the same. Construct is a new workflow, which is called as a reusable workflow from Upload. I mean having it as an additional job in workflow Build, without creating a new (reusable) workflow.
The Build workflow might be executed directly, not as a reusable workflow, which would ignore the recipes/jobs defined in Construct. However, if you put this job in Build, then it will always be run together with other recipes.

Reply:
> Not the same. Construct is a new workflow, which is called as a reusable workflow from Upload. I mean having it as an additional job in workflow Build, without creating a new (reusable) workflow. The Build workflow might be executed directly, not as a reusable workflow, which would ignore the recipes/jobs defined in Construct. However, if you put this job in Build, then it will always be run together with other recipes.

Good point that it shouldn't be in fact run from `Upload` but from `Build`. Perhaps you could move the Construct job then @proppy and use `needs: "openlane-linux"` for it? It'd be also nice to have `openlane.sky130a` parametrized in Construct.yml so that it's reusable if we ever want to have more installers constructed.

Reply:
> The Build workflow might be executed directly, not as a reusable workflow, which would ignore the recipes/jobs defined in Construct. However, if you put this job in Build, then it will always be run together with other recipes.

My thinking that whe only want to "release" the installer when we actually publish the packages (so w/ that logic the same way we don't run `Upload` when we run `Build` independently, we couldn't run `Construct`).